core/register_command.php

```diff
@@ -193,6 +193,8 @@
 	$application->add(new OC\Core\Command\User\ListCommand(\OC::$server->getUserManager(), \OC::$server->getGroupManager()));
 	$application->add(new OC\Core\Command\User\Info(\OC::$server->getUserManager(), \OC::$server->getGroupManager()));
 	$application->add(new OC\Core\Command\User\AddAppPassword(\OC::$server->get(\OCP\IUserManager::class), \OC::$server->get(\OC\Authentication\Token\IProvider::class), \OC::$server->get(\OCP\Security\ISecureRandom::class), \OC::$server->get(\OCP\EventDispatcher\IEventDispatcher::class)));
+	$application->add(new OC\Core\Command\User\AuthTokens(\OC::$server->get(\OCP\IUserManager::class), \OC::$server->get(\OC\Authentication\Token\IProvider::class)));
+	$application->add(new OC\Core\Command\User\DeleteAuthToken(\OC::$server->get(\OC\Authentication\Token\IProvider::class)));
```

Comment:
Since all parameters are DI-able we can also just use:
```suggestion
	$application->add(\OC::$server->get(\OC\Core\Command\User\AuthTokens::class));
	$application->add(\OC::$server->get(\OC\Core\Command\User\DeleteAuthToken::class));
```

Reply:
Oh I see, nice